### DISCUSSION TITLE
manually select gradient scheme for column in JupyterLab

### DISCUSSION THREAD

**Discussion by zxweed** [perspective-dev/perspective#1863] [Q&A] [answered]
## Support Question

Hello, how can I specify in the PerspectiveWidget constructor call that I want to use a gradient (or bar) view scheme for certain columns?

**Comment by timkpaine** [comment, marked as answer]
Gradient:
`plugin_config={'columns': {'YOUR COLUMN NAME': {'gradient': MAX VALUE FOR GRADIENT, 'number_color_mode': 'gradient'}}}`

Bar
`plugin_config={'columns': {'YOUR COLUMN NAME': {'gradient': MAX VALUE FOR GRADIENT, 'number_color_mode': 'bar'}}}`
